src/osdc/ObjectCacher.h

```diff
@@ -345,8 +350,9 @@ class ObjectCacher {
                  map<loff_t, BufferHead*>& hits,
                  map<loff_t, BufferHead*>& missing,
                  map<loff_t, BufferHead*>& rx,
-		 map<loff_t, BufferHead*>& errors);
-    BufferHead *map_write(ObjectExtent &ex, ceph_tid_t tid);
+		 map<loff_t, BufferHead*>& errors,
+      ZTracer::Trace *trace = nullptr);
```

Comment:
Minor: why default the ```trace``` param to nullptr? At this stage, I'd expect it already be set to a valid trace or to nullptr. Likewise, why not pass the trace to ```map_read``` from within the ```_readx``` method? It would be nice to consolidate the trace to only ```Bufferhead``` if possible.